index.html

```diff
@@ -111,6 +119,9 @@ <h1 class="post-title2">
                     <h1 class="post-title2">
                         Profesional Art Printing Data
                     </h1>
+                    <h1 class="hide-for-mobile">
```

Comment:
- [ ] As written in the project requirements. I kindly suggest that you ensure that when a project is hovered on with the mouse:
-  the description and dark opacity disappear
-  and a See Project button appears to the bottom of the image and takes the whole image width.
-   Your `See Project` button should link to the Live Demo of the project or to the GitHub repository.

- [ ] [OPTIONAL] There should be only one `h1` in an HTML file because of semantics. I kindly suggest that you make only your main title of `"Hey There.I’m Yaseen"` an `h1` and kindly make use of headers from `h2` down to `h6` for any other headers in your HTML file.
- [ ] [OPTIONAL] please kindly make changes and replace all other `h1` except your main title of "Hey There.I’m Yaseen"

Reply:
ok, sir but the images of the projects are dark in nature how to remove the darkness and make it bright while the image I downloaded from Figma is dark by default can I use the bright image of the first project for other projects when they are hoovered on  Mr. @topeogunleye

Reply:
> * 
> *

Replied on slack :+1: